### DISCUSSION TITLE
Module images

### DISCUSSION THREAD

**Discussion by ciroiriarte** [netbox-community/devicetype-library#1344] [Q&A]
Would it be ok to contribute module images?. I'm working with network cards for PowerEdge servers.

**Comment by danner26** [comment]
Hey @ciroiriarte images currently are only for devices, not for modules. Network cards for servers are technically modules so these would not be included

**Comment by ciroiriarte** [comment]
Hello!,

There is a feature to add an image to the module type, would you consider
accepting them in a separate folder/directory as contribution?.

It really helps for local inspection and planning when who design/plans is
different from who implements.

We're happily using it.

Regards,
CI._

**Reply by danner26** [reply]
Hey @ciroiriarte, as long as the functionality is added in the upstream NetBox repository then we would be happy to adapt

**Reply by ciroiriarte** [reply]
Hey!, Today (v3.3.0), it's possible to add module type images. That's all, adding reference images for whoever is working in the DC.

![image](https://github.com/netbox-community/devicetype-library/assets/1750260/b25d889b-08c1-430f-8135-7dbab78cc5a0)

I'm still struggling with my first PR though, hope to find some time this week to fix it.

**Reply by danner26** [reply]
Hey @ciroiriarte I have added the ability to add module images now. If you sync your fork with the upstream master, you will now see a module-images folder. Images should have the same name as the device type and should preferably be in png format
